custom_visibility/src/custom/data/visibility/FilterOpportunities.php

```diff
@@ -168,23 +187,42 @@ protected function getTableAlias()
     }
 
     /**
-     * {@inheritdoc}
+     * implementation of ElasticStrategyInterface interface
+     *
+     * to create ealstic analyzer for the fields introduced here,
+     * leave the function empty if you don't need special analyzer
```

Comment:
lines 192 and 193 don't feel like a complete thought.  If they are separate thoughts, we should end 192 with a period.  I'm wondering if the word "to" on line 192 shouldn't be there?